### DISCUSSION TITLE
Excuse me, how to get the address of trx

### DISCUSSION THREAD

**Discussion by elvis126** [consenlabs/webview#60] [General]
I have switched my wallet to TRX wallet, go to the "Browse" page to visit

const provider = new ethers.providers.Web3Provider(web3.currentProvider);
const signer = provider.getSigner();
const address = await signer.getAddress();

But the obtained address is still eth, not trx.

Excuse me, how to get the address of trx

**Comment by xwartz** [comment]
You should use the [tronweb](https://github.com/tronprotocol/tronweb).

```
const address = tronWeb.defaultAddress
```

**Reply by elvis126** [reply]
How imToken wallet authorizes the acquisition of trx address ？
